config/bundles.php

```diff
@@ -5,4 +5,6 @@
 return [
     Libero\ContentPageBundle\ContentPageBundle::class => ['all' => true],
     Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
+    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
+    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
```

Comment:
why in `test`? Is this just the toolbar, wouldn't expect PHPUnit to use it?

Reply:
Not needed at the moment, can be used to test things like sent emails (eg https://github.com/elifesciences/journal/blob/238014172fea7fc4c385f782e531f9f96c762708/features/bootstrap/Context.php#L82-L108).